server/controllers/userRegister.js

```diff
@@ -0,0 +1,14 @@
+const { postCreateUser } = require('../database/postQueries/postCreateUser');
+const { hashPassword } = require('../utils/hashPassword');
+const { createJWT } = require('../utils/jwtHelper');
+
+exports.post = (req, res) => {
+  const { password } = req.body;
+  hashPassword(password)
+    .then(hash => postCreateUser({ ...req.body, password: hash }))
+    .then((userId) => {
+      res.cookie('jwt', createJWT(userId), { maxAge: 1000 * 60 * 60 * 30, httpOnly: true });
```

Comment:
Why are you setting JWT in 2 files? you can register and redirect to login again so it would set in 1 place 
This is more of an enhancement you don't have to change it

Reply:
Refactored the code so it is only written once.